src/schema/fair.ts

```diff
@@ -47,6 +49,41 @@ const FairType = new GraphQLObjectType({
   name: "Fair",
   fields: () => ({
     ...GravityIDFields,
+    artists: {
+      type: artistConnection,
+      args: pageable({
+        sort: {
+          type: FairArtistSortsType,
+          description: "Sorts for artists in a fair",
+        },
+      }),
+      resolve: (
+        { id },
+        options,
+        _request,
+        { rootValue: { fairArtistsLoader, artistsLoader } }
+      ) => {
+        const gravityOptions = omit(
+          convertConnectionArgsToGravityArgs(options),
+          ["page"]
+        )
+
+        gravityOptions.total_count = true
```

Comment:
👍

Reply:
I don't think so - I think it just exposes an `X-Total-Count` as a response header, though not sure the mechanism of how gravity arrives at that count. Refactored from a separate call to retrieve the count after @mzikherman's feedback here: https://github.com/artsy/metaphysics/pull/1430#discussion_r241569306